vmdb/spec/models/ems_refresh/refreshers/openshift_refresher_spec.rb

```diff
@@ -0,0 +1,109 @@
+require "spec_helper"
+
+describe EmsRefresh::Refreshers::OpenshiftRefresher do
+  before(:each) do
+    @ems = FactoryGirl.create(:ems_openshift, :hostname => "10.35.0.167")
+  end
+
+  it "will perform a full refresh on openshift" do
+    VCR.use_cassette("#{described_class.name.underscore}") do #, :record => :new_episodes) do
```

Comment:
> Please be sure the VCR tests are wrapped in 2.times like the other VCR tests. This ensures that a second pass with the exact same data doesn't create duplicate records.

OK.
I see that not all refreshers are having that, for example, the rhev one - doesn't.
Also, some refreshers do "@ems.reload" twice inside the loop, but for example foreman one doesn't - can you explain why/what is the correct one in openshift/k8s case?